usage/app/lib/Config.scala

```diff
@@ -32,6 +33,7 @@ object Config extends CommonPlayAppProperties with CommonPlayAppConfig {
   val capiPreviewUser = properties("capi.preview.user")
   val capiPreviewPassword = properties("capi.preview.password")
   val capiPageSize = Try(properties("capi.page.size").toInt).getOrElse[Int](defaultPageSize)
+  val capiMaxRetries = Try(properties("capi.maxRetries").toInt).getOrElse[Int](defaultMaxRetries)
```

Comment:
I'm not sure if it's worth it. This allows for a default (set above) so i'm not sure if it's worth actually setting it.

This pattern allows us to override this value manually with config if we ever need to (like locally).